guides/source/action_view_overview.md

```diff
@@ -243,14 +243,14 @@ You can also do:
 By default `ActionView::Partials::PartialRenderer` has its object in a local variable with the same name as the template. So, given:
 
 ```erb
-<%= render partial: "product" %>
+<%= render @product %>
 ```
 
 within `_product` partial we'll get `@product` in the local variable `product`,
 as if we had written:
 
 ```erb
-<%= render partial: "product", locals: { product: @product } %>
+<%= render partial: "prodcuts/product", locals: { product: @product } %>
```

Comment:
👋 Hi ! You have a little typo I guess here 🙈

Reply:
Oh, right 🙈 I was staring at it for 2 minutes until I realized what you are talking about 😄 Let me fix it!